src/Components/CreateImageWizardV2/steps/TargetEnvironment/Azure/AzureTarget.tsx

```diff
@@ -0,0 +1,312 @@
+ * to azure.
+ * @param manual set to true if the user will enter the information manually, set to
+ * false if they are fetched from the sources
+ * @param setManual a function to update the `manual` value
+ * @param sourceId if not in manual mode holds the selected source by
+ * the user
+ * @param setsourceId a function to update the sourceId
+ * @param tenantId a value either computed via the sources if a sourceId
+ * was selected or manually entered by the user
+ * @param setTenantId a function to update the tenantId
+ * @param subscriptionId a value either computed via the sources if a sourceId
+ * was selected or manually entered by the user
+ * @param setSubscriptionId a function to update the subscriptionId
+ * @param resourceGroup holds a resource group the user either selected
+ * in a set of possibilities or entered manually
+ * @param setResourceGroup a function to update the
+ * ResourceGroup
+ * @param resourceGroups a list of resource groups computed via the sources if a
+ * sourceId was selected that contains choices for the user to select
+ * the resourceGroup.
+ */
+const AzureTarget = ({
+  manual,
+  setManual,
+  isErrorFetchingDetails,
+  sourceId,
+  setSourceId,
+  tenantId,
+  setTenantId,
+  subscriptionId,
+  setSubscriptionId,
+  resourceGroup,
+  setResourceGroup,
+  resourceGroups,
+}: AzureTargetPropTypes) => {
+  const [isErrorFetchingSources, setIsErrorFetchingSources] = useState(false);
+  const clearState = () => {
+    // when the user switches from manual to automatic, reset everything
+    setSourceId(0);
+    setTenantId('');
+    setResourceGroup('');
+    setSubscriptionId('');
+  };
+
+  return (
+    <>
+      <Form>
+        <Title headingLevel="h2">Target environment - Microsoft Azure</Title>
+        <TextContent>
+          <Text>
+            Upon build, Image Builder sends the image to the selected authorized
+            Azure account. The image will be uploaded to the resource group in
+            the subscription you specify.
+          </Text>
+          <Text>
+            To authorize Image Builder to push images to Microsoft Azure, the
+            account owner must configure Image Builder as an authorized
+            application for a specific tenant ID and give it the role of
+            &quot;Contributor&quot; for the resource group you want to upload
+            to. This applies even when defining target by Source selection.
+            <br />
+            <Button
+              component="a"
+              target="_blank"
+              variant="link"
+              icon={<ExternalLinkAltIcon />}
+              iconPosition="right"
+              isInline
+              href="https://docs.microsoft.com/en-us/azure/active-directory/develop/v2-oauth2-auth-code-flow"
+            >
+              Learn more about OAuth 2.0
+            </Button>
+          </Text>
+        </TextContent>
+        <FormGroup label="Share method:" isRequired>
```

Comment:
Share method does not need to be required as there is no way to disable both radios

```suggestion
        <FormGroup label="Share method:">
```

original:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/ca304bf9-c797-4d8b-972e-4cd033b32df8)


current:
![image](https://github.com/RedHatInsights/image-builder-frontend/assets/49452678/9f0e87d9-7df4-4051-b6a8-7fefd46675db)